src/js/utils.js

```diff
@@ -54,3 +54,35 @@ export function updateCookieExpiration(cookieName, newExpiration) {
   // Re-set cookie with same values and new expiration date
   Cookies.set(cookieName, existingValue, { expires: newExpiration });
 }
+
+// change date into how long ago that date is relative to when view page
+export function dateHowLongAgo(date) {
```

Comment:
sorta feeling like this file is long enough to be broken into several files under a util folder.

it's RIGHT about at that cusp which I'd start considering. up to you but def should break it out if add any more than this.